src/components/index/page-sections/CallToActionSection.jsx

```diff
@@ -0,0 +1,44 @@
+import { Link } from '@reach/router';
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import CallToActionLogo from '../../../images/call-to-action.png';
+import IndexGradientBorderButtonBasic from '../../buttons/gradient-border-buttons/IndexGradientBorderButtonBasic';
+// import PropTypes from 'prop-types';
```

Comment:
Hier (und in den nächsten Komponenten) auch einfach raus mit den imports. Den größten Teil davon wirst hier ja eh nicht wieder brauchen (scroll bar etc)